en/releases/main.md

```diff
@@ -29,7 +29,9 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 
 ### Common
 
-- TBD
```

Comment:
FYI, I have added this release not entry to "main". The first one :-). PX4 v1.15 docs have been split off.

Reply:
Thats perfect, thanks!